samples/apps/http-apps/Dockerfiles/device

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.15 as build
+
+ARG PROJECT="http-extensibility"
+ARG MODULE="github.com/deislabs/akri/${PROJECT}"
+
+WORKDIR /${PROJECT}
+
+# Copy go.mod first and install dependencies
+COPY go.mod .
+RUN go mod download
+
+# Copy all sources
+COPY . .
+
+# Compile Go binary
+RUN GOOS=linux \
+    go build -a -installsuffix cgo \
+    -o /bin/device \
+    ${MODULE}/cmd/device
+
+
+FROM gcr.io/distroless/base-debian10
+
+COPY --from=build /bin/device /
```

Comment:
should `/bin/device` be `/bin/${PROJECT}`?

Reply:
maybe something like this, where `ARGS="--path=/ --path=/sensor"` and `COMPONENT=device`:

```sh
CMD ["sh", "-c", "/${COMPONENT} ${ARGS}"]
```